src/Plugin/KeyInput/ApigeeAuthKeyInput.php

```diff
@@ -266,6 +288,9 @@ public function processSubmittedKeyValue(FormStateInterface $form_state) {
         $input_values['authorization_server'] = '';
         $input_values['client_id'] = '';
         $input_values['client_secret'] = '';
+        if ($input_values['gcp_hosted'] === '1') {
+          $input_values['account_json_key'] = '';
+        }
```

Comment:
This will never be true, because it's within an _if_ that only evaluates if `$input_values['gcp_hosted']` is unset or false. We may need to move this logic outside this scope.

Reply:
I have addressed this in the recent code push